### DISCUSSION TITLE
Looking for a django-ninja tutorial

### DISCUSSION THREAD

**Discussion by josephgu41** [vitalik/django-ninja#802] [General]
​I'm looking for some hands-on guidance on developing applications with django-ninja. I don't know how to use it to develop a complete application that can be used in production. There are many practical tutorials on drf on youtube, but I didn't find a good tutorial on using ninja. Do you have any good recommendations? Any videos and articles are fine. Thanks.

**Comment by vitalik** [comment]
you can find few here

https://django-ninja.rest-framework.com/tutorial/other/video/

**Reply by josephgu41** [reply]
Roger that!Didn't notice this video when reading the document, thank you!I appreciate your framework. I think it's easier for me to use and more modern than drf.Wish Django ninja better and better!
